Intersect.Server/Entities/Player.cs

```diff
@@ -1786,6 +1786,8 @@ public bool TryGiveItem(Item item, ItemHandling handler = ItemHandling.Normal, b
 
             // Get this information so we can use it later.
             var openSlots = FindOpenInventorySlots().Count;
```

Comment:
How come this is cast to a double?

Reply:
`Math.Ceiling` requires a double, and integer division returns an integer